scripts/scripts.js

```diff
@@ -128,6 +146,25 @@ export function decorateButtons(element) {
   });
 }
 
+/**
+ * When there are 2 buttons in a row, display them next to each other.
+ */
+
+export function groupMultipleButtons(main) {
+  const buttons = main.querySelectorAll('p.button-container');
+  buttons.forEach((button) => {
+    console.log(button);
```

Comment:
You can go ahead and remove this console statement.

Reply:
This is done.